src/Starcounter.VS.CSBoilerplateApplicationProjectTemplate/MyTemplate.vstemplate

```diff
@@ -0,0 +1,46 @@
+<VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
+  <TemplateData>
+    <Name>Starcounter Boilerplate Application</Name>
+    <Description>A boilerplate project for creating a C# application that executes inside Starcounter</Description>
```

Comment:
In VS most project templates use 'Blank' as a prefix for a bare minimum project. I'm thinking we could maybe use that on the already existing project to match the convention in VS.

Then we could maybe use the 'Standard' or 'Basic' prefix for this template and adapt the descriptions there after.